services/s3/src/test/java/software/amazon/awssdk/services/s3/S3PresignerTest.java

```diff
@@ -227,6 +228,7 @@ public void getObject_Sigv4PresignerHonorsSignatureDuration() {
 
         assertThat(presigned.httpRequest().rawQueryParameters().get("X-Amz-Expires").get(0)).satisfies(expires -> {
             assertThat(expires).containsOnlyDigits();
+            // TODO(sra-identity-auth): This should be isEqualTo(1234)?
```

Comment:
I think it's a bug if isCloseTo was needed for some reason earlier instead of isEqualTo check. Not sure why it was so from the beginning, but couldn't change it right now because of derived attributes behavior for PRESIGNER_EXPIRATION. I have an internal discussion thread about it. Will update that separately.

Reply:
Addressing the derived attributes and isCloseTo in https://github.com/aws/aws-sdk-java-v2/pull/4602